components/organisms/o-header.vue

```diff
@@ -63,6 +66,9 @@ export default {
   methods: {
     categoryLink (category) {
       return formatCategoryLink(category);
+    },
+    isCategoryActive (category) {
+      return this.getCurrentCategory.path ? this.getCurrentCategory.path.startsWith(category.path) : false;
```

Comment:
@psmyrek This is a good solution for default configuration but this can not work in real projects. In Magento you can set in URL rewrite rules which are considered by VueStorefront. You can set for example path `test` for parent category and `other-slug` for child category. In these cases, this solution will not work. 

Maybe solution based on `categoryId` and `parentId` will be good here (I think we should check it for two levels parent->child->grandchild)

Reply:
@qiqqq The `path` property in category object is not an URL (string) that we see in adress bar, but `path` property is a chain of category IDs from root to current one.

Example:
for URL `/gear/gear-3` (Gear main menu) the `path` property is `1/2/3`
for URL `/gear/bags/bags-4` (Gear/Bags sub-menu) the `path` property is `1/2/3/4`

I think current solution is exactly what you proposed.

Reply:
Yes, you are right. I thought `path` is the URL part. Great